sites/public/styles/blocks.scss

```diff
@@ -25,6 +25,18 @@
   font-weight: 600;
 }
 
+.action-block.no-results {
+  .action-block__header {
+    font-size: var(--bloom-font-size-lg);
+  }
+  .action-block__icon {
+    margin-bottom: var(--bloom-s8);
+  }
+  .action-block__subheader {
+    padding-top: var(--bloom-s4);
```

Comment:
I'm curious how important these CSS changes are. For the most part they aren't drastically changing the appearance of action block. Just slightly smaller header and larger spacing between the sections. Ideally we wouldn't want this custom of CSS if we don't have to. And if this is the standard spacing going forward we should think about updating the ActionBlock component instead so it gets applied everywhere. 

That does increase the scope though and I'm fine with these changes if we believe this is needed and should be separate from the other ActionBlock usages.

One thing to consider though is it appears that the header for ActionBlock normally changes size depending on the width of the screen. Is that something we would want to persist for this use case?

Reply:
@ludtkemorgan This all makes sense to me and I share your concerns on the custom CSS. It seems like this is a call for @slowbot. Do you think the customizations in the Figma are critical or should I stick to the general Action Block spacing? For comparison, the screenshot below is the Action Block without customizations.
<img width="691" alt="Screen Shot 2022-10-21 at 9 34 55 AM" src="https://user-images.githubusercontent.com/53269332/197245626-3c5ae821-e127-4899-b0c7-265fac54e197.png">